### DISCUSSION TITLE
How to install flutterflow-ui in a nylo project.?

### DISCUSSION THREAD

**Discussion by geonidas6** [nylo-core/nylo#125] [Ideas]
It would be really great to be able to create the design with flutterflow and integrate it into a nylo project. But currently there are package incompatibility errors

**Comment by agordn52** [comment]
Hi @geonidas6,

Thanks for the idea, currently Nylo isn't compatible with FlutterFlow.
It's an interesting thought, I think it would only be possible with a collaboration between Nylo and FlutterFlow.

**Reply by geonidas6** [reply]
but can you make Nylo compatible with 
flutterflow_ui

https://pub.dev/packages/flutterflow_ui

**Reply by agordn52** [reply]
Thanks for the idea but it's not so easy, if there's a real demand for FlutterFlow x Nylo then maybe I'll delve into it.
